components/CheckBox/CheckBox.tsx

```diff
@@ -8,19 +8,25 @@ const { Panel } = Collapse;
 
 function CheckBox({ filters, handleFilters }) {
   const [Checked, setChecked] = React.useState<number[]>([]);
+
   React.useEffect(() => {
     const stored: number[] = JSON.parse(localStorage.getItem("checked")) ?? [];
     setChecked(stored);
     handleFilters(stored);
   }, []);
 
+  React.useEffect(() => {
```

Comment:
I'd suggest some changes, like:
for what is this useEffect being used for?

if it is not used, you can delete it.

Reply:
True, it's removed ;)